src/cascadia/PublicTerminalCore/PublicTerminalCore.vcxproj

```diff
@@ -46,12 +43,13 @@
       <Project>{48d21369-3d7b-4431-9967-24e81292cf62}</Project>
     </ProjectReference>
   </ItemGroup>
-
   <ItemDefinitionGroup>
     <ClCompile>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
     </ClCompile>
+    <Link>
+      <AdditionalDependencies>d3dcompiler.lib</AdditionalDependencies>

```

Comment:
nit: You may just want to add this to the common link line in `common.pre.props`